sky/packages/sky/lib/src/fn3/homogeneous_viewport.dart

```diff
@@ -96,7 +96,7 @@ class HomogeneousViewportElement extends RenderObjectElement<HomogeneousViewport
     // assertions since you can't normally mutate the render object tree during layout. (If there was
```

Comment:
reword the comment to talk about starting a build scope rather than locking the framework state.